### PR TITLE
RESTWS-805: Constructor should not be used to instantiate "String"

### PR DESCRIPTION
**Pull request by theanandankit**
## Issue I worked on

see https://issues.openmrs.org/browse/RESTWS-805

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Review by HerbertYiga — APPROVED** [review]

File: omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java [hunk 1]
```diff
@@ -369,7 +369,7 @@ private boolean testOperationImplemented(OperationEnum operation, DelegatingReso
 					if (method == null) {
 						return false;
 					} else {
-						method.invoke(resourceHandler, RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID, new String(),
+						method.invoke(resourceHandler, RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID,"",
```
Comment (sacull) [inline omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java:372]:
```suggestion
						method.invoke(resourceHandler, RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID, "",
```

File: omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java [hunk 2]
```diff
@@ -382,7 +382,7 @@ private boolean testOperationImplemented(OperationEnum operation, DelegatingReso
 						return false;
 					} else {
 						method.invoke(resourceHandler, RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID,
-						    RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID, new String(), new RequestContext());
+						    RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID,"", new RequestContext());
```
Comment (sacull) [inline omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java:385]:
```suggestion
						    RestConstants.SWAGGER_IMPOSSIBLE_UNIQUE_ID, "", new RequestContext());
```
Yes, I know, I'm picking on details. ;P

**Comment by theanandankit** [comment]
@sacull i have changed the code as you suggested.

**Review by sacull — APPROVED** [review]
It's good for me now. ;)

**Pull request merged**